profile/profile.yaml

```diff
@@ -0,0 +1,110 @@
+# Preferred name (Characters only)
+name: "Miguel Dario"
+# Your GitHub repo root URL
+github_repo_root_url: https://github.com/jarviscanada/jarvis_data_eng_MiguelDario
+
+# Introduce yourself (80-120 words)
+# Suggestions: you should 
+# 1) describe your academic and work experience (please focus on transferable/job related tech and soft skills only)
+# 2) explain your passion for the role you are looking for (e.g. developer, BSA, QA, tech support, etc.)
+# 3) mention what kind of role you are looking for
+# 4) briefly talk about your qualifications (soft and tech) to justify why the hiring manager should hire your 
+summary: My name is Miguel Dario, and I Graduated from the Business Technology Management program at Ryerson University, where I was exposed to fields such as data analytics, project management, and networking. I have previously worked as an intern for Manawa Networks where I developed strong client management skills through providing IT solutions. In the same job, I was responsible for documenting various IT systems and standards. I have a passion for technology and enjoy learning about new technologies and how to make sense of it. I also enjoy interacting with a wide range of clients, learning about their business problems, and providing solutions to those through technology. I believe that these combined make me a great fit for a Business Systems Analyst role.
+
+skills:
+  # Please include 
+  #     1)soft skills (e.g. elicitation, communicaiton, collaberation, interpersonal, etc ) 
+  #     2) business tools (such as Excel, Viso, etc.)
+  Business Skills:
+    - Documentation
+    - Client Management
+    - Presentation
+    - Requirements Elicitation
+    - Agile/Scrum
+  # Please add more data related skills, such as Python, Tableau, R, Matlab, AI, ML, Statistics, etc.
+  # You can also add some programming skills, such as programming languages, GIT, Linux, Cloud Platform, tools, etc.
+  Technical Skills:
+    - Data Analytics
```

Comment:
Data Analytics with Python, Tableau, and R

Reply:
Visio

Reply:
Done.